[BUG]: runtime error: invalid memory address or nil pointer dereference

run into this issue and cant say what this issue caused:
```
21:15:28 map:merger           [DEBUG] | merging key=name from fr******e -> fre*********r
21:15:28 map:merger           [DEBUG] | merging key=name from fr******e -> fre*********h
21:15:28 map:merger           [DEBUG] | merging key=username from fre********R -> fre********H
21:15:28 map:merger           [DEBUG] | merging key=password from Supe*************rd -> $n3********z
21:15:28 map:merger           [DEBUG] | merging key=name from fr******e -> fre*********dt
21:15:28 map:merger           [DEBUG] | merging key=password from Supe*************rd -> $n3********z
21:15:28 map:merger           [DEBUG] | merging key=username from fre********R -> fre*********T
21:15:28 map:merger           [DEBUG] | merging key=url from http:*****************080 -> http:/*****************080
21:15:28 map:merger           [DEBUG] | merging key=name from fr******e -> fre*********dt
21:15:28 map:merger           [DEBUG] | merging key=username from fre********R -> hy*****R
21:15:28 map:merger           [DEBUG] | merging key=url from http:*****************080 -> http:/*****************8080
21:15:28 map:merger           [DEBUG] | merging key=password from Supe*************rd -> $n3********z
21:15:28 map:merger           [DEBUG] | merging key=url from http:*****************080 -> http:/******************8080
21:15:28 map:merger           [DEBUG] | merging key=username from fre********R -> hy******T
21:15:28 map:merger           [DEBUG] | merging key=password from Supe*************rd -> $n3********z
21:15:28 map:merger           [DEBUG] | merging key=name from fr******e -> fre*********dt
21:15:28 command:default      [INFO ] | [with-cache] Connection cluster '1A'
  http     - http://freqtrade-eur:8080 'freqtradeEUR:Supe*************rd'
  database - disabled
21:15:28 command:default      [INFO ] | [with-cache] Connection cluster '2A'
  http     - http://freqtrade-eth:8080 'freqtradeETH:$n3********z'
  database - disabled
21:15:28 command:default      [INFO ] | [with-cache] Connection cluster '3A'
  http     - http://freqtrade-usdt:8080 'freqtradeUSDT:$n3********z'
  database - disabled
21:15:28 command:default      [INFO ] | [with-cache] Connection cluster '4A'
  http     - http://freqtrade-bteur:8080 'hyperEUR:$n3********z'
  database - disabled
21:15:28 command:default      [INFO ] | [with-cache] Connection cluster '5A'
  http     - http://freqtrade-btusdt:8080 'hyperUSDT:$n3********z'
  database - disabled
21:15:28 client:http:1A       [DEBUG] | request: GET http://freqtrade-eur:8080/api/v1/ping
21:15:28 cache:service        [DEBUG] | creating 'http.total-call.1A' data
21:15:28 cache:service        [DEBUG] | creating 'http.success-call.1A' data

panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x18 pc=0x88e2f2]

goroutine 1 [running]:
github.com/kamontat/fthelper/metric/v4/src/clients.(*Http).handleError(0xc0002097a0, 0x0, 0x0, 0xafe8e0, 0xc00000e190)
        /home/runner/work/fthelper/fthelper/metric/src/clients/http.go:39 +0x52
github.com/kamontat/fthelper/metric/v4/src/clients.(*Http).Initial(0xc0002097a0, 0x80, 0x80)
        /home/runner/work/fthelper/fthelper/metric/src/clients/http.go:45 +0xb0
github.com/kamontat/fthelper/metric/v4/src/freqtrade.(*Freqtrade).Initial(0xc00029f170, 0xc0002a2100, 0xc000298b20)
        /home/runner/work/fthelper/fthelper/metric/src/freqtrade/freqtrade.go:29 +0x47
github.com/kamontat/fthelper/metric/v4/src/connection.(*cache).Initial(0xc0002a0040, 0xc0002a0100, 0x4)
        /home/runner/work/fthelper/fthelper/metric/src/connection/cache.go:33 +0x76
main.defaultCommand(0xc00001f9a0, 0x203000, 0xc00002b000)
        /home/runner/work/fthelper/fthelper/metric/main.go:52 +0x3b3
github.com/kamontat/fthelper/shared/commandline/commands.(*Command).Start(0xc00002a360, 0xc00001f9a0, 0x2, 0xc00002b000)
        /home/runner/work/fthelper/fthelper/shared/commandline/commands/command.go:27 +0x5a
github.com/kamontat/fthelper/shared/commandline.(*cli).Start(0xc000020540, 0xc0000b0000, 0x5, 0x5, 0xc000066568, 0xc000050930)
        /home/runner/work/fthelper/fthelper/shared/commandline/cli.go:87 +0x9b2
main.main()
        /home/runner/work/fthelper/fthelper/metric/main.go:97 +0x477
```

--- a/metric/src/clients/http.go
+++ b/metric/src/clients/http.go
@@ -35,8 +35,10 @@ func (c *Http) handleError(err error) error {
 		return err
 	}
 
-	c.Enabled = false
-	c.logger.Error(err.Error())
+	if err != nil {
+		c.Enabled = false
+		c.logger.Error(err.Error())
+	}
 	return nil
 }
 

